src/main/groovy/org/transmartproject/batch/highdim/datastd/ChromosomalRegionValidator.groovy

```diff
@@ -21,11 +27,15 @@ trait ChromosomalRegionValidator implements Validator {
         assert target instanceof ChromosomalRegionSupport
 
         String chromosome = target.chromosome?.trim()
-        if (optionalDefinition
-                && !chromosome && target.startBp == null && target.endBp == null) {
+        def chrInfoSpecified =  chromosome || target.startBp != null || target.endBp != null
+        if (optionalDefinition && !chrInfoSpecified) {
             return
```

Comment:
@cataphract yes, that's the idea. There would be nothing to validate if all fields are empty.